hs_core/tasks.py

```diff
@@ -301,17 +299,26 @@ def manage_task_hourly():
                 create_bag_by_irods(res.short_id)
             else:
                 # retry of metadata deposition failed again, notify admin
-                msg_lst.append("Metadata deposition with CrossRef for the published resource "
-                               "DOI {res_doi} failed again after retry with first metadata "
-                               "deposition requested since {pub_date}.".format(res_doi=act_doi,
-                                                                               pub_date=pub_date))
+                if CrossRefSubmissionStatus.UPDATE_FAILURE not in res.doi:
+                    # this is the case of retry of initial deposit (deposit when publishing the resource)
+                    msg = f"Metadata deposition with CrossRef for the published resource " \
+                          f"DOI {act_doi} failed again after retry with first metadata " \
+                          f"deposition requested since {pub_date}."
+                else:
+                    # this is the case of updating crossref metadata deposit as a result of resource metadata change
+                    # subsequent to the initial deposit
+                    msg = f"Metadata UPDATE deposition with CrossRef for the published resource " \
+                          f"DOI {act_doi} failed again after retry."
+
+                msg_lst.append(msg)
                 logger.debug(response.content)
         else:
             msg_lst.append("{res_id} does not have published date in its metadata.".format(
                 res_id=res.short_id))
 
+    # get all published resources with doi ending with 'pending' or 'update_pending'
     pending_resources = BaseResource.objects.filter(raccess__published=True,
-                                                    doi__contains='pending')
+                                                    doi__endswith=CrossRefSubmissionStatus.PENDING.value)
```

Comment:
It has to be `CrossRefSubmissionStatus.PENDING.value` when using in DB query. Otherwise, instead of using 'pending' the query looks for string 'CrossRefSubmissionStatus.PENDING'. I found that out the hard way last Friday!

Reply:
wow. okay thanks didn't know that